SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -247,6 +273,7 @@ - (void)onSessionInitializationCompleteForSession:(SDLIAPSession *)session {
     // Data Session Opened
     if (![controlProtocolString isEqualToString:session.protocol]) {
         self.sessionSetupInProgress = NO;
+        [self sdl_backgroundTaskEnd];
```

Comment:
I think you are correct, but I think the better place for this would be in the setter for `sessionSetupInProgress`. If it goes to `NO`, then the task must end. I say this to make sure you hit all cases, because I believe there may be some cases where the accessory fails to connect and we don't complete the task. Unless that is desired to attempt to connect to other accessories?